From be328c224dea1375078091cfdb375d219f32a3b1 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Mon, 2 Jul 2018 09:53:55 +0800
Subject: [PATCH] arm64: dts: rockchip: default disable cpu opp-1200000000 for
 rk3308

According to P45 of <<Rockchip RK3308 Datasheet V1.0-20180313>>,
the max voltage for Logic is 1.1V, so for many boards which
Core_VDD and Logic_VDD share the same power supply, the ARM
core voltage will be limited to 1.1V, that can only guarantee
the cpu to run at the max freq of 1008MHZ.

As for board which Core_VDD and Logic_VDD use independent
power supply, the cpu can run a higher freq up to 1296MHZ
with higher Core_VDD power supply, you can eanble it in
your board dts.

Change-Id: I69777aa5bee797e609d9ea122f3502347930b631
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308.dtsi b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
index b8bee600686b..276247d21db9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
@@ -126,6 +126,7 @@
 			opp-hz = /bits/ 64 <1200000000>;
 			opp-microvolt = <1200000 1200000 1340000>;
 			clock-latency-ns = <40000>;
+			status = "disabled";
 		};
 		opp-1296000000 {
 			opp-hz = /bits/ 64 <1296000000>;
-- 
2.35.3

